streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -80,8 +88,14 @@ public TopicPartition partition() {
      * @return the size of this queue
      */
     public int addRawRecords(Iterable<ConsumerRecord<byte[], byte[]>> rawRecords) {
+        ConsumerRecord<Object, Object> record = null;
```

Comment:
I don't see any reason for this variable to not be declared inside the loop as it was previously